platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -515,19 +513,6 @@ public SwirldsPlatform(@NonNull final PlatformComponentBuilder builder) {
             eventStreamManagerName = String.valueOf(selfId);
         }
 
-        final EventStreamManager eventStreamManager = new DefaultEventStreamManager(
-                platformContext,
-                time,
-                threadManager,
-                getSelfId(),
-                this,
-                eventStreamManagerName,
-                eventConfig.enableEventStreaming(),
-                eventConfig.eventsLogDir(),
-                eventConfig.eventsLogPeriod(),
-                eventConfig.eventStreamQueueCapacity(),
-                this::isLastEventBeforeRestart);
```

Comment:
`SwirldsPlatform::isLastEventBeforeRestart` is now unused

Reply:
Nice! Deleted.